bin/dummy_mentor

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+questionnaire=$1
+default_page=index.html
+last_page=final.html
+questionnaire_page=questionnaire.html
+
+cd public/dummy
+
+rm -rf mentor
+wget \
+		 --save-cookies /tmp/cookies.txt \
+     --keep-session-cookies \
+     --recursive \
+     --no-clobber \
+     --page-requisites \
+     --html-extension \
+     --domains vsv.io \
+     --restrict-file-names=unix \
+     --no-parent \
+		 -E -H -k -K -p \
+     -r -e robots=off \
+     -U mozilla \
+    http://vsv.io\?q\=$questionnaire
```

Comment:
willen we vsv.io ook in een variabele zetten? misschien wel zo netjes

Reply:
done